front/app/modules/commercial/content_builder/admin/components/CraftComponents/AboutBox/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+// components
+import ProjectInfoSideBar from 'containers/ProjectsShowPage/shared/header/ProjectInfoSideBar';
+
+// hooks
+import useProject from 'hooks/useProject';
+
+// utils
+import { isNil } from 'lodash-es';
+
+const AboutBox = ({ projectId }: { projectId: string }) => {
+  const project = useProject({ projectId });
+  return <ProjectInfoSideBar projectId={isNil(project) ? '' : project.id} />;
```

Comment:
Why not just pass the `projectId` directly to the component? No need to have `useProject` here.